tools/segemehl/segemehl.xml

```diff
@@ -137,47 +151,57 @@
           <param name="first_strand_query" type="data" format="fastqsanger,fastqillumina,fastq,fasta" label="Reads from first strand" />
           <param name="second_strand_query" type="data" format="fastqsanger,fastqillumina,fastq,fasta" label="Reads from second strand" />
         </repeat>
-        <param name="maxinsertsize" type="integer" value="5000" label="Maximum size of the inserts (paired end)" help="default: 5000 (-I)" />
+        <param name="maxinsertsize" argument="-I/--maxinsertsize" type="integer" value="5000" label="Maximum size of the inserts (paired end)" help="default: 5000" />
```

Comment:
The argument is meant to be just one param. In this case you could just use argument="--maxinsertsize" and remove the `name="maxinsertsize"` altogether.

Reply:
don